docs/api/algorand/create-delegated-payment.mdx

```diff
@@ -0,0 +1,105 @@
+# CreateDelegatedPayment
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Request Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- | 
+| assetId         | `string` | The ID of the DeCash asset that is being sent. |
+| sender.address  | `string` | The address of the account that is sending the funds. Note that this account must have enough funds to cover both the amount that is being sent, and the [fee](#CreateDelegatedPayment). |
+| sender.amount   | `number` | The amount that is being sent. |
+| receiver        | `string` | The address of the account that is receiving the funds. Note that, if this account is not a [DelegatedWallet](#CreateDelegatedPayment), then it must be opted into the token that is being sent. | 
+
+
+
+## Responses
+
+### Regular response
+
+| Name | Type | Description |
+| ---- | ---- | ----------- | 
+| noSign | `string[]` | The array of transactions (in msgpack) that shall not be signed. Together with toSign it composes the full array of transactions in the group. |
+| toSign | `string[]` | The array of transactions (in msgpack) that must be signed with the spending key of the `sender` account. Together with the noSign transactions it composes the full array of transactions in the group. |
+| info.fee | `number` | Amount of DeCash that is being spent on fees in this transaction. | 
```

Comment:
Could be `info` and declare type as object `{fee: number}`. However, I am okay with this too.